docs/nightly/en/user-guide/ingest-data/for-iot/sql.md

```diff
@@ -0,0 +1,114 @@
+# SQL
+
+We will use the `monitor` table as an example to show how to write data.
+
```

Comment:
We need to mention MySQL and PostgreSQL compatibility here. For example, how to use those database drivers to connect to the database